sale_discount_display_amount/__manifest__.py

```diff
@@ -6,16 +6,18 @@
     'summary': """
         This addon intends to display the amount of the discount computed on
         sale_order_line and sale_order level""",
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
No need to change this, will be done on merge

Reply:
ok, will do